src/nnet3/nnet-example-utils.cc

```diff
@@ -192,5 +192,200 @@ void GetComputationRequest(const Nnet &nnet,
     KALDI_ERR << "No outputs in computation request.";
 }
 
+int32 NumFramesPerChunk(const NnetIo &io) {
```

Comment:
Please add a few lines of comments describing what each method does, for all the methods in all files.